front/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
+    "downlevelIteration": true,
```

Comment:
Do you still use this?

Reply:
yes, for formatting links I use a for / of loop on an array